libultraship/libultraship/Lib/ImGui/imgui.h

```diff
@@ -1622,6 +1622,7 @@ enum ImGuiCol_
     ImGuiCol_ScrollbarGrabHovered,
     ImGuiCol_ScrollbarGrabActive,
     ImGuiCol_CheckMark,
+    ImGuiCol_Cross,
```

Comment:
so what do we do about the case where checking one thing forces another checkbox to be true?

we'd still want to disable the checkbox, but we wouldn't want to have an `X` there as it'd imply it's set to false

Reply:
Me and leggettc18 were talking about this on discord as well. Ideally if we implement setting CVars to a temporary value (when a save is loaded with race settings for example), we need to be able to set the graphic to match the temporary value.

For now, I could add another argument to the checkbox that decides what graphic to draw when it's disabled. So something like `int disabledGraphic` and have none, checkmark and cross as options.

Reply:
i like that idea, probably would be good to use an enum instead of an int to avoid magic numbers

Reply:
Sorry yeah, with none, checkmark and cross in my previous comment. I meant enums with names similar to that. Will push the changes soon.